src/features/events/index.js

```diff
@@ -1,4 +1,5 @@
 export { default as EventListingCard } from './components/eventListingCard'
+export { default as GroupedEventsCards } from './components/groupedEventsCard'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There should be no space before '}'.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)